File: app/src/main/java/com/justcode/hxl/viewutil/自定义View/绘制1_1/饼图/interfaces/PieTextFactory.java
package com.justcode.hxl.viewutil.自定义View.绘制1_1.饼图.interfaces;


import com.justcode.hxl.viewutil.自定义View.绘制1_1.饼图.drawable.BaseTextDrawable;

public interface PieTextFactory<T extends BaseTextDrawable> {
    T createPieText();
}

File: app/src/main/java/com/justcode/hxl/viewutil/自定义View/绘制1_1/饼图/interface/PieTextFactory.kt
package com.justcode.hxl.viewutil.自定义View.绘制1_1.饼图.`interface`

import com.justcode.hxl.viewutil.自定义View.绘制1_1.饼图.drawable.BaseTextDrawable

interface PieTextFactory <T : BaseTextDrawable>{
    fun createPieText(): T
}